Frontend/ui-common/src/FileProviders/ConfigProvider/ConfigProvider.purs

```diff
@@ -60,7 +61,7 @@ loadAppConfig _ =
 
 getConfigFromFile :: String -> Array Foreign
 getConfigFromFile fileName = do
-  let config = loadFileInDUI $ fileName <> ReExport.dotJSA
+  let config = loadFileInDUI $ fileName <> ReExport.dotJS
```

Comment:
is this change done intentionally?

Reply:
Yes, while developing locally, the configuration was being taken from .jsa rather than local v1-configuration.js file. That's why updated this. But need to debug this issue.